crates/brix_processor/src/lib.rs

```diff
@@ -1,14 +1,20 @@
-use tinytemplate::TinyTemplate;
-
-mod context;
-use context::Context;
+use handlebars::Handlebars;
+use serde_json::json;
+use serde_json::value::{Map, Value as Json};
+use std::collections::HashMap;
 
 use brix_errors::BrixError;
 
-pub fn process(module: String, content: String) -> Result<String, BrixError> {
-    let mut tt = TinyTemplate::new();
-    tt.add_template(&module, &content)?;
-
-    let result = tt.render(&module, &Context::new(module.clone()))?;
+pub fn process(text: String, context: Map<String, Json>) -> Result<String, BrixError> {
+    let handlebars = Handlebars::new();
+    let result = handlebars.render_template(&text, &context)?;
     Ok(result)
 }
+
+pub fn create_context(data: HashMap<String, String>) -> Map<String, Json> {
+    let mut res = Map::new();
+    for (key, value) in data.into_iter() {
+        res.insert(key, json!(value));
```

Comment:
what are you doing here and why?

Reply:
This just converts an std HashMap into a JSON one so that the user doesn't have to import serde_json into their crate.

Reply:
right, but why are you converting it to json? we're not outputting json anywhere

Reply:
handlebars (the templating engine we're using) seems to require JSON for context.